workspaces/ui-v2/src/pages/onboarding/Onboarding.tsx

```diff
@@ -0,0 +1,202 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Page } from '<src>/components';
+import { useAppSelector } from '<src>/store';
+import { useAppConfig } from '<src>/contexts/config/AppConfiguration';
+import { useAsyncMemo } from 'use-async-memo';
+import { Checkbox, Typography } from '@material-ui/core';
+import {
+  CapturesServiceContext,
+  useCaptures,
+} from '<src>/hooks/useCapturesHook';
+
+const useStyles = makeStyles((theme) => ({
+  body: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    width: '60%',
+    marginTop: theme.spacing(2),
+  },
+  root: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+}));
+
+interface OnboardingData {
+  specs_shared_with_team: number;
+  specs_shared_not_with_team: number;
+  specs_uploaded_by_gitbot: number;
+  coverages_uploaded: number;
+}
+
+export default function Onboarding() {
```

Comment:
Also minor (and perhaps this is more personal preference) - I don't really like default exports, it makes it hard to trace usage (if you are using search) - IDEs can provide tools to help with this, but this makes it easy to trace where these are used.

Also you can type this as `export const Onboarding: FC = () => {...}`